libs/core/src/lib/message-strip/message-strip.component.ts

```diff
@@ -99,6 +107,14 @@ export class MessageStripComponent implements OnInit, OnChanges, CssClassBuilder
         this.buildComponentCssClass();
     }
 
+    /** @hidden */
+    ngAfterViewInit(): void {
+        if (!this.ariaLabel) {
+            this.ariaLabel = this.messageStripContent.nativeElement.innerText;
```

Comment:
Just a question. What will happen if the user passes a more complex content like text+button or text+link or even another component?

Reply:
hmm, not sure about this. How do you think we should handle this then?

Reply:
<img width="1384" alt="Screen Shot 2021-05-26 at 1 31 01 PM" src="https://user-images.githubusercontent.com/39598672/119705504-d2d2d800-be26-11eb-928d-4918e8b46bca.png">

I tested this with adding a button to the text. It takes the content of the button and append it to the other text.

Reply:
@mikerodonnell89 FYI